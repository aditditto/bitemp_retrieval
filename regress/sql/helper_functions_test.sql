DROP EXTENSION bitemp_retrieval;
CREATE EXTENSION bitemp_retrieval;
SET DATESTYLE TO 'ISO';
SET TIMEZONE TO 'Asia/Jakarta';

SELECT interval_len('["2005-01-01","2006-05-02")'::tstzrange);