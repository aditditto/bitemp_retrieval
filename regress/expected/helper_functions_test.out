DROP EXTENSION bitemp_retrieval;
ERROR:  extension "bitemp_retrieval" does not exist
CREATE EXTENSION bitemp_retrieval;
SET DATESTYLE TO 'ISO';
SET TIMEZONE TO 'Asia/Jakarta';
SELECT interval_len('["2005-01-01","2006-05-02")'::tstzrange);
     interval_len      
-----------------------
 @ 1 year 4 mons 1 day
(1 row)

